tests/ui/print_stderr.rs

```diff
@@ -0,0 +1,6 @@
+#![warn(clippy::print_stderr)]
+
+fn main() {
+    eprintln!("Hello");
+    eprint!("World");
```

Comment:
Let's add some negative tests using `println!()` and `print!()` to see they don't trigger the lint